packages/browsers/src/CLI.ts

```diff
@@ -61,7 +61,8 @@ export class CLI {
 
   #defineBrowserParameter(yargs: yargs.Argv<unknown>): void {
     yargs.positional('browser', {
-      description: 'The browser version',
+      description:
+        'Which browser to install <browser>[@<buildId|latest>]. `latest` will try to find the latest available build. `buildId` is a browser-specific identifier such as a version or a revision.',
```

Comment:
This command is used for both `install` and `launch` but the second does not have a `latest`. Maybe we should input the description as an argument?

Reply:
This is listed in known limitations. Eventually, we would probably support latest in both.